src/app/calculator/utilities/power-factor-correction/power-factor-correction-form/power-factor-correction-form.component.html

```diff
@@ -80,37 +94,48 @@
     </th>
 
     <th>
-      <span *ngIf="(data.billedForDemand == 0 && data.adjustedOrActual == 0) || (data.billedForDemand == 0 && data.adjustedOrActual == 1) || (data.billedForDemand == 1 && data.adjustedOrActual == 0)">
+      <span *ngIf="(data.billedForDemand == 0 && data.adjustedOrActual == 0) || (data.billedForDemand == 0 && data.adjustedOrActual == 1) || (data.billedForDemand == 0 && data.adjustedOrActual == 2) || (data.billedForDemand == 1 && data.adjustedOrActual == 0)">
         Power Factor
       </span>
       <span *ngIf="data.billedForDemand == 1 && data.adjustedOrActual == 1">
         Real Demand (kW)
       </span>
     </th>
+    <th *ngIf="data.billedForDemand == 0 && data.adjustedOrActual == 2">
+      <span>
+        Actual Demand (kW)
+      </span>
+    </th>
     <th>
 
     </th>
   </tr>
 
   <tr *ngFor="let input of data.monthyInputs; let i = index;">
     <td>
-      Month {{i + 1}}
+      <input name="{{'month_'+i}}" type="text" step="any"
+        [(ngModel)]="input.month" id="{{'month_'+i}}" (input)="calculate()" onfocus="this.select();">
     </td>
     <td>
-      <input name="{{'input1_'+i}}" class="form-control" type="number" step="any"
+      <input name="{{'input1_'+i}}" type="number" step="any"
         [(ngModel)]="input.input1" id="{{'input1_'+i}}" (input)="calculate()" onfocus="this.select();"
         (focus)="focusField('input1')">      
     </td>
     <td>
-      <input name="{{'input2_'+i}}" class="form-control" type="number" step="any"
+      <input name="{{'input2_'+i}}" type="number" step="any"
         [(ngModel)]="input.input2" id="{{'input2_'+i}}" (input)="calculate()" onfocus="this.select();"
         (focus)="focusField('input2')">   
     </td>
-    <td class="btn-group btn-group-sm">
-      <button type="button" class="btn btn-danger pull-right btn-sm btn-form" (click)="btnDeleteMonth(i)">
+    <td *ngIf="data.billedForDemand == 0 && data.adjustedOrActual == 2">
+      <input name="{{'input3_'+i}}" type="number" step="any"
+        [(ngModel)]="input.input3" id="{{'input3_'+i}}" (input)="calculate()" onfocus="this.select();"
+        (focus)="focusField('input3')">   
+    </td>
+    <td class="btn-group btn-group-sm" *ngIf="i+1 == data.monthyInputs.length">
```

Comment:
Just a note. Not required change. I believe this is a check for if it's the last item in an ngFor. In an *ngFor you could do... 

`*ngFor="let input of data.monthyInputs; let i = index; let last = last;"` and then this would be `*ngIf="last"`